### ISSUE TITLE
Split payment collection options 

### ISSUE THREAD

**Issue by getinfo-developer** [Attendize/Attendize#462]
Hi,  Want to know if currently Attendize support split payment collection option in more than one account. e.g we want to setup two paypal payment option. and split the payment either in absolute value or in percentage basis, it like Paypal account 1 will get 95% and Paypal account 2 will get 5% of the total order amount.

**Comment by vitorpamplona** [comment]
2. Can be solved with Feature Request #542 (Stripe Connect)

**Comment by clawfire** [comment]
Yep. And now with the Stripe Checkout SCA support, it's even easier to add.
